### ISSUE TITLE
Support for Excel day count as input

### ISSUE THREAD

**Issue by bobjansen** [eddelbuettel/anytime#17]
Excel stores dates internally as the number of days since 1899-12-30, i.e. today is 42664. Importing Excel files doesn't always work perfectly and it can happen that these dates are not recognized as dates by the import functionality. In that scenario, I'd like to convert these numbers using `anytime`.

Is support for this date format wanted? Could Excel type dates overlap with other formats?

**Comment by eddelbuettel** [comment]
As a rule, the package will always opt for "sane" over "insane" -- and if in doubt, lean towards ISO.  

Hence parsing of 20161021 as YYYYMMDD which is the most common variant.   

Here, we have now way to differentiate the _integer_ values given in that format and the integer values given by a normal origin, ie 17095.   So no.

In fact, right now we ignore _both_ as integer (and numeric) values that low are rejected so that something like 20161021 can in fact be parsed.   

So in short we cannot for a) unusual / not sensitive origin and b) we currently don't do low-valued integers.

I plan to close this.

**Comment by bobjansen** [comment]
No problem, I understand.

**Comment by eddelbuettel** [comment]
Thanks for your understanding.
